packages/terra-show-hide/src/_ShowHideButton.module.scss

```diff
@@ -0,0 +1,102 @@
+@import '~terra-mixins';
+
+:local {
+  // html input[type='button'].terra-Button selector used to
+  // prevent a WebKit bug destroys native `audio` and `video` controls
+  // in Android 4. More info:
+  // https://github.com/necolas/normalize.css/issues/20
+  // https://code.google.com/p/android/issues/detail?id=34301
+  // appearance: button; set to correct the inability to style
+  // clickable types in iOS and Safari.
+  // Appearance needs to be applied only to button/input elements, as
+  // applying to anchor element will cause render issue.
+  // More info:
+  // https://github.com/necolas/normalize.css/issues/566
+  // https://github.com/necolas/normalize.css/commit/1be44f3b2a79af5d463c7ea0264fee0e6fa464ca
+  // stylelint-disable selector-no-qualifying-type
+  button.button,
+  html input[type='button'].button,
+  input[type='reset'].button,
+  input[type='submit'].button {
+    appearance: button;  // autoprefix fix into -webkit-appearance
+  }
+  // stylelint-enable selector-no-qualifying-type
+
+  .button {
+    background-color: transparent; // Remove the gray background on active links in IE 10
+    border: 0;
+    color: var(--terra-show-hide-color, #0059a8);
+    cursor: pointer;
+    display: inline-block; // prevents variant icon from breaking onto its own line
+    font-family: inherit;
+    font-size: var(--terra-show-hide-font-size, 1rem);
+    line-height: var(--terra-show-hide-line-height, 1.429);
```

Comment:
lets make a little taller to be more touch-friendly:
`line-height: var(--terra-show-hide-line-height, 1.85);`

Reply:
https://github.com/cerner/terra-core/pull/1956/commits/c898c0ea7a377cb2138c6673068b9f61b7237808